### ISSUE TITLE
Image files not being displayed

### ISSUE THREAD

**Issue by paulvonber** [directus/directus#21387]
### Describe the Bug

After upgrade to v10.9.0 got bunch of errors and none of the images being returned/displayed
<img width="1602" alt="image" src="https://github.com/directus/directus/assets/37245111/9f31a1e1-7856-4308-b1ce-292835709540">

Would appreciate any suggestions.

### To Reproduce

Upgrade to v10.9.0

### Directus Version

v10.9.0

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by paescuj** [comment]
Can you share your configuration?

**Comment by paulvonber** [comment]
@paescuj here it is, I have IMPORT_IP_DENY_LIST commented out, but if I uncomment it things doesn't change
<img width="898" alt="image" src="https://github.com/directus/directus/assets/37245111/29f51b2b-4eda-4455-bbd5-47c3f8954885">

**Comment by paescuj** [comment]
It seems like you have a Flow in place which runs on `read` event for the `directus_files` collection and does an external request, is that correct?
Due to the same underlining issue as reported in #21351, requests in Flows might not work correctly at the moment.
We're working on a fix for this issue, which will be available in the next release.
In the meantime you might want to (partially) disable the Flow or downgrade to 10.8.3.

**Comment by vanling** [comment]
@paescuj same issue "Location "undefined" doesn't exist." On multiple images from s3

Happened directly after updating docker setup to `image: directus/directus:10.9.0`

I do not have any flows setup.

```
{
    "errors": [
        {
            "message": "An unexpected error occurred.",
            "extensions": {
                "code": "INTERNAL_SERVER_ERROR"
            }
        }
    ]
}
```

**Comment by paulvonber** [comment]
@paescuj I don't actually have flow related to "directus_files", but I do have one for "directus_activity" if that anyhow can help you.

**Comment by vanling** [comment]
🤔 might this be related? https://github.com/directus/directus/issues/21359

**Comment by arturoguzmanp** [comment]
Just weighing in I am having the same issue after updating to 10.9 - cluster pm2 non-docker version

**Comment by rijkvanzanten** [comment]
@vanling I think you're right, but we'll keep this open until a patch release early next week so we can confirm 👍🏻

**Comment by cliqer** [comment]
I have the same problem reading from local

**Comment by ElliotHYLee** [comment]
not just images, files as well

**Comment by tzdesign** [comment]
Is the password field also crashing if you create a new user?

**Comment by abdonrd** [comment]
I need to add at least this fields:

<img width="861" alt="Screenshot 2024-02-11 at 23 07 15" src="https://github.com/directus/directus/assets/1007051/485ab0ee-b9a5-465c-981b-38ec4967350a">

**Comment by paescuj** [comment]
@tzdesign 
> Is the password field also crashing if you create a new user?

Sounds like you're affected by this issue: #21369
This has been fixed in #21384 which will be available in the next release.

**Comment by paescuj** [comment]
I can confirm this is fixed by #21360 which has already been merged and will be available in the coming release.

**Comment by paescuj** [comment]
The fix (#21360) has been released as part of [v10.9.1](https://github.com/directus/directus/releases/tag/v10.9.1).
